### PR TITLE
remove rules conflicting with prettier

### PR DESCRIPTION
**Pull request by robrichard**

### CODE REVIEW

**Comment by infinityfarm** [comment]
```The following rules are unnecessary or might conflict with Prettier:

- array-bracket-spacing
- comma-dangle
- comma-spacing
- comma-style
- eol-last
- flowtype/space-after-type-colon
- flowtype/union-intersection-spacing
- func-call-spacing
- indent
- key-spacing
- keyword-spacing
- no-extra-semi
- no-mixed-spaces-and-tabs
- no-multi-spaces
- no-spaced-func
- no-trailing-spaces
- react/jsx-closing-bracket-location
- react/jsx-indent-props
- react/jsx-wrap-multilines
- semi
- space-before-blocks
- space-before-function-paren
- space-in-parens
- space-infix-ops
- space-unary-ops
- wrap-iife

The following rules are enabled but cannot be automatically checked. See:
https://github.com/prettier/eslint-config-prettier#special-rules

- no-mixed-operators
- no-unexpected-multiline```

**Comment by robrichard** [comment]
@infinityfarm I think I got them all

File: packages/eslint-config-1stdibs-base/rules/style.js [hunk 1]
```diff
@@ -65,50 +16,14 @@ module.exports = {
         "no-bitwise": 2,
         "no-lonely-if": 1,
         "no-mixed-operators": 1,
```
Comment (infinityfarm) [inline packages/eslint-config-1stdibs-base/rules/style.js]: @robrichard I think you should remove "no-mixed-operators" here. See: https://github.com/prettier/eslint-config-prettier#no-mixed-operators

**Comment by infinityfarm** [comment]
@robrichard looks like there's a "no-unexpected-multiline" in `packages/eslint-config-1stdibs-base/rules/errors.js` that should go as well.

**Comment by infinityfarm** [comment]
@robrichard It says those two rules "require special attention when writing code" in order to not conflict, which doesn't sound cool

**Comment by robrichard** [comment]
@infinityfarm done

**Pull request merged**